Parser/convertSBEcnvVar.m

```diff
@@ -122,12 +122,21 @@
     case 'flC'
       name = 'CPHL';
       comment = 'Artificial chlorophyll data computed from bio-optical sensor raw counts measurements. Originally expressed in ug/l, 1l = 0.001m3 was assumed.';
-      
+
+    %Aquatracka SBE19plus new chla #550
+    case 'flCUVA'
+      name = 'CPHL';
+      comment = 'Chlorophyll estimated from fluorescence. No coeficients provided.  Chelsea UV Aquatracka assumed in ug/l, from SBE19Plus.';
```

Comment:
I changed the comments for two reasons:

1. convertSBEcnvVar is used by both SBE37 and SBE19. This is why I put SBE19Plus in the comment - to differentiate in the case where SBE37 has flCUVA. If this is not possible in any case, then I should remove; otherwise, the removal can create provenance tracking problems.

2. There is no way to differentiate "flC" of "flCUVA". They probably are measured by different sensors. I got no clue of what the respective sensor for "flC" from the code. I would like to know when concatenating CPHL that they came from different sensors and who is who. The comments are not the optimal way, but ATM is the only way to do that.

If I copy the comment, we lost provenance of "Aquatracka" sensor.  

I can add the units as comments ( which should be read but this is another story). However,  I want to keep the reference of the issue since this is extra functionality (and is easy to grep).